db/blip_handler.go

```diff
@@ -322,7 +322,8 @@ func (bh *blipHandler) sendChanges(sender *blip.Sender, opts *sendChangesOptions
 
 					// If change is a removal and we're running with protocol V3 and change change is not a tombstone
 					// fall into 3.0 removal handling
-					if change.allRemoved && bh.blipContext.ActiveProtocol() == BlipCBMobileReplicationV3 && !change.Deleted {
+					if change.allRemoved && bh.blipContext.ActiveSubprotocol() == BlipCBMobileReplicationV3 && !change.
+						Deleted {
```

Comment:
This linebreak looks unexpected/weird to me, even if it's valid syntax. We're not too strict on line lengths to warrant breaking it up

```suggestion
					if change.allRemoved && bh.blipContext.ActiveSubprotocol() == BlipCBMobileReplicationV3 && !change.Deleted {
```

Reply:
Makes sense, this was an accidental formatting. Removed in the latest revision.